### PR TITLE
remove unused 'total_seen_options' variable

### PR DESCRIPTION
**Pull request by fichtner**
Signed-off-by: Franco Fichtner <franco@opnsense.org>

### CODE REVIEW

**Review by melroy89 — APPROVED** [review]
Remove dead code is always good. Thx.

**Comment by stappersg** [comment]
On Mon, Dec 09, 2024 at 05:42:51AM -0800, Melroy van den Berg wrote:
> @melroy89 approved this pull request.

:-)

 
> Remove dead code is always good. Thx. 

Yes.  However my approval waits for a `Sign-of-by: `.
(And the real reason for insisting on the Sign-of
is my wish for having it on first version of future
contributions to radvd  :-)
See also https://github.com/radvd-project/radvd/pull/193#issuecomment-1296286565
and https://github.com/radvd-project/radvd/pull/234#issuecomment-2525214929


Groeten
Geert Stappers
-- 
Silence is hard to parse

**Comment by melroy89** [comment]
> Yes. However my approval waits for a `Sign-of-by: `. (And the real reason for insisting on the Sign-of is my wish for having it on first version of future contributions to radvd :-) See also [#193 (comment)](https://github.com/radvd-project/radvd/pull/193#issuecomment-1296286565) and [#234 (comment)](https://github.com/radvd-project/radvd/pull/234#issuecomment-2525214929)

Ah indeed. It's a good practice to configure this by default in `.gitconfig`.. Add `signingkey` via: `git config --global user.signingkey <your_key>`.

I also auto-sign commits (`commit.gpgsign` config).

More info: https://docs.github.com/en/authentication/managing-commit-signature-verification/telling-git-about-your-signing-key

**Comment by fichtner** [comment]
@stappersg personally I'm not a fan of `--signoff` but I've amended the commit to reflect the requirement

**Review by stappersg — APPROVED** [review]
Thank you
